plugins/mobile-plugin/src/test/java/pagefactory/mobile/pages/auth/EmailPage.java

```diff
@@ -0,0 +1,29 @@
+package pagefactory.mobile.pages.auth;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.pagefactory.mobile.MobilePage;
+import ru.sbtqa.tag.pagefactory.Page;
+import ru.sbtqa.tag.pagefactory.annotations.ElementTitle;
+import ru.sbtqa.tag.pagefactory.annotations.PageEntry;
+
+@PageEntry(title = "Страница авторизации. Ввод email")
+public class EmailPage extends MobilePage {
+
+    private static final Logger LOG = LoggerFactory.getLogger(Page.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)